src/patterns/components/navigation/narrow-navigation.hbs

```diff
@@ -5,65 +5,58 @@ description: |
+        <span class="sprk-b-TypeBodyTwo sprk-c-Accordion__heading {{heading}}">
+          Item 1
+        </span>
+
+        <svg class="sprk-c-Icon sprk-c-Accordion__icon" data-sprk-toggle="icon" viewBox="0 0 448 512">
+          <use xlink:href="#chevron-down"></use>
+        </svg>
       </a>
-      <ul class="sprk-c-NarrowNavigation sprk-c-NarrowNavigation--sub sprk-u-HideWhenJS" data-sprk-toggle="content">
-        <li class="sprk-c-NarrowNavigation__sub-item">
-          <a class="sprk-c-NarrowNavigation__link sprk-c-NarrowNavigation__link--sub" href="">Item 1</a>
+      <ul class="sprk-b-List sprk-b-List--bare sprk-c-Accordion__details sprk-u-HideWhenJS" data-sprk-toggle="content">
+        <li>
+          <a class="sprk-b-Link sprk-b-Link--standalone sprk-u-pam" href="">Item 1</a>
         </li>
-        <li class="sprk-c-NarrowNavigation__sub-item">
-          <a class="sprk-c-NarrowNavigation__link sprk-c-NarrowNavigation__link--sub" href="">Item 2</a>
+        <li>
+          <a class="sprk-b-Link sprk-b-Link--standalone sprk-u-pam" href="">Item 2</a>
         </li>
-        <li class="sprk-c-NarrowNavigation__sub-item">
-          <a class="sprk-c-NarrowNavigation__link sprk-c-NarrowNavigation__link--sub" href="">Item 3</a>
+        <li>
+          <a class="sprk-b-Link sprk-b-Link--standalone sprk-u-pam" href="">Item 3</a>
         </li>
       </ul>
     </li>
-    <li class="sprk-c-NarrowNavigation__item">
-      <a class="sprk-c-NarrowNavigation__link " href="">
-        <div class="sprk-o-SideBySide">
-          <div class="sprk-o-SideBySide__item sprk-o-SideBySide__item--three-quarters">
-            <span> Item 2 </span>
-          </div>
-          <div class="sprk-o-SideBySide__item sprk-o-SideBySide__item--one-quarter"></div>
-        </div>
+
+    <li class="sprk-c-Accordion__item {{item}}">
+      <a aria-controls="details1" class="sprk-c-Accordion__summary {{summary}}" href="#">
+        <span class="sprk-b-TypeBodyTwo sprk-c-Accordion__heading {{heading}}">
+          Item 2
+        </span>
       </a>
     </li>
-    <li class="sprk-c-NarrowNavigation__item" data-sprk-toggle="container">
-      <a class="sprk-c-NarrowNavigation__link " href="" data-sprk-toggle="trigger">
-        <div class="sprk-o-SideBySide">
-          <div class="sprk-o-SideBySide__item sprk-o-SideBySide__item--three-quarters">
-            <span>Item 3</span>
-          </div>
-          <div class="sprk-o-SideBySide__item sprk-o-SideBySide__item--one-quarter">
-            <svg class="sprk-c-Icon sprk-c-NarrowNavigation__arrow" viewBox="0 0 100 100" data-sprk-toggle="icon">
-              <use xlink:href="#chevron-down" />
-            </svg>
-          </div>
-        </div>
+
+    <li class="sprk-c-Accordion__item {{item}}" data-sprk-toggle="container">
+      <a aria-controls="details1" class="sprk-c-Accordion__summary {{summary}}" data-sprk-toggle="trigger" data-sprk-toggle-type="accordion" href="#">
```

Comment:
Something we should figure out in the future just to clean the code blocks up in the docs is the {{vars}} that we use. Right now if the component doesn't have that set then it leaves an extra white space in our doc output like `class="sprk-c-Accordion__summary  "` which, isn't the biggest deal, of course. But, it is something we can avoid. But, would take writing extra code. I think I did it for one component by using if statements to check if it's there. But, I didn't do it for every component I've wrote. Anyway! Thought I'd bring it up.